src/theme/components/Layout.tsx

```diff
@@ -61,20 +61,20 @@ const Layout = ({ children }: Props): JSX.Element => (
           enableDeclineButton={true}
```

Comment:
To fix the accessibility error of the contrast of compatible colors  : https://dequeuniversity.com/rules/axe/4.6/color-contrast-enhanced